jupyterlab_gcedetails/src/components/hardware_scaling_form.tsx

```diff
@@ -200,11 +202,37 @@ export class HardwareScalingForm extends React.Component<Props, State> {
     this.props.onSubmit(configuration);
   }
 
+  private displayPricingEstimation() {
+    const { details, priceService } = this.props;
+    const { configuration } = this.state;
+
+    const zone = details.instance.zone.split('/').pop();
```

Comment:
Just out of curiosity, I know we use this `split('/').pop()` convention in quite a few places, do you think it would make sense to create a helper function for it?